actions/DisplayAction.php

```diff
@@ -197,13 +197,16 @@ private function createFeedItemFromException($e, BridgeInterface $bridge): FeedI
         $item = new FeedItem();
 
         // Create "new" error message every 24 hours
-        $_error_time = urlencode((int)(time() / 86400));
-        $itemTitle = sprintf('Bridge returned error %s! (%s)', $e->getCode(), $_error_time);
+        $dailyUid = urlencode((int)(time() / 86400));
+        $itemTitle = sprintf('Bridge returned error %s! (%s)', $e->getCode(), $dailyUid);
         $item->setTitle($itemTitle);
         $item->setURI(get_current_url());
         $item->setTimestamp(time());
 
-        // todo: consider giving more helpful error messages
+        // Create a unique id for feed readers e.g. "staysafetv twitch videos_2023_19389"
+        $itemUid = $bridge->getName() . '_' . date('Y') . '_' . $dailyUid;
+        $item->setUid($itemUid);
```

Comment:
The magic happens here.

Reply:
Do we need date('Y') here? $dailyUid already "contains" year.

Reply:
Ah yes you are correct.